test/unit/utils/test-xhr-utils.js

```diff
@@ -202,4 +203,72 @@ describes.sandboxed('utils/xhr-utils', {}, env => {
     });
   });
 
+  describe('getViewerAuthTokenIfAvailable', () => {
```

Comment:
Thanks for adding these!

Reply:
No problem! I'm glad the tests finally got setup!